LondonBritishLibrary/orient/BLorient13262.xml

```diff
@@ -0,0 +1,174 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient13262" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Malkǝʾa Dāwit, Magical Prayers</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>BL Oriental 13262</idno>
+                        <altIdentifier><idno>Or. 13262</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 45</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>  
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r" to="35v"/>
+                            <title ref="LIT3052RepCh332"/>
+                            <incipit xml:lang="gez">እምሥርወ፡ ዕሤይ፡ እስመ፡ ሠረፅከ፡ ወእማኅፀና፡ ለሩት።</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="35v"/>
+                            <title ref="LIT7002PrayerSmiths"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ ሕማመ፡ ቡዳ፡ ወጠቢብ፡ ሕማመ፡ ቡዳ፡ ሐሮፓሮስ፡ ፯ጊዜ፡</incipit>
```

Comment:
Sometimes there is excessive spaces between signs/elements of encoding; I think if left like this, they will be also visualised like unnecessary double spaces.